Listenbrainz/Application/ListenbrainzApp.swift

```diff
@@ -33,8 +33,14 @@ struct ListenbrainzApp: App {
           //.environmentObject(spotifyManager)
             .onAppear {
               //handleSpotifySession()
-              homeViewModel.requestMusicData(userName: userName)
-              feedViewModel.fetchFeedEvents(username: userName, userToken: userToken)
+              Task {
```

Comment:
@gauravbhardwaj7 remove from here as well and add to FeedView

Reply:
@akshaaatt  done!